packages/devextreme/testing/testcafe/tests/accessibility/splitter.ts

```diff
@@ -0,0 +1,87 @@
+import url from '../../helpers/getPageUrl';
+import { clearTestPage } from '../../helpers/clearPage';
+import { testAccessibility, Configuration } from '../../helpers/accessibility/test';
+import { Options } from '../../helpers/generateOptionMatrix';
+import { Item, Properties } from '../../../../js/ui/splitter.d';
+
+fixture.disablePageReloads`Accessibility`
+  .page(url(__dirname, '../container.html'))
+  .afterEach(async () => clearTestPage());
+
+const items: Item[] = [
+  {
+    resizable: true,
+    minSize: '70px',
+    size: '140px',
+    text: 'Left Pane',
+  }, {
+    splitter: {
+      orientation: 'vertical',
+      items: [
+        {
+          resizable: true,
+          collapsible: true,
+          maxSize: '75%',
+          text: 'Central Top Pane',
+        }, {
+          collapsible: true,
+          splitter: {
+            orientation: 'horizontal',
+            items: [
+              {
+                resizable: false,
+                collapsible: true,
+                size: '30%',
+                minSize: '5%',
+                text: 'Nested Left Pane',
+              }, {
+                collapsible: true,
+                resizable: true,
+                text: 'Nested Central Pane',
+              }, {
+                resizable: true,
+                collapsible: true,
+                collapsed: true,
+                size: '30%',
+                minSize: '5%',
+                text: 'Nested Right Pane',
+              },
+            ],
+          },
+        },
+      ],
+    },
+  }, {
+    size: '140px',
+    resizable: false,
+    collapsible: false,
+    text: 'Right Pane',
+  },
+];
+
+const options: Options<Properties> = {
+  dataSource: [items],
+  allowKeyboardNavigation: [true, false],
+  disabled: [true, false],
+  width: [450, 'auto', '100%'],
+  height: [400],
+  separatorSize: [8, 5],
+};
+
+const a11yCheckConfig = {
+  // NOTE: color-contrast issues
```

Comment:
Let's update the note and remove extra comments with aria-orientation and 'color-contrast'. I suggest you to add aria-orientation comment to the task card and attach a link to failed jobs.

Reply:
https://github.com/DevExpress/DevExtreme/pull/26984/commits/6aabf515713fd30266eab7540fd6850ab77521c6